v2_1/ws/rest/docs/4_6_1_other_queries.md

```diff
@@ -2,76 +2,79 @@
+agencyID | A string compliant with the SDMX *common:NCNameIDType* | The agency maintaining the artefact for which data have been reported. 
+resourceID | A string compliant with the SDMX *common:IDType* | The id of the artefact for which data have been reported.
+version | A string compliant with the *VersionType* defined in the SDMX Open API specification | The version of the artefact for which data have been reported.
+key | A string compliant with the *KeyType* defined in the SDMX Open API specification. | The combination of dimension values identifying the slice of the cube for which data should be returned. Wildcarding is supported via the `*` operator. For example, if the following key identifies the bilateral exchange rates for the daily US dollar exchange rate against the euro, D.USD.EUR.SP00.A, then the following key can be used to retrieve the data for all currencies against the euro: D.*.EUR.SP00.A.
 componentId | A string compliant with the SDMX common: IDType | The id of the Dimension for which to obtain availability information about.  In the case where this information is not provided, data availability will be provided for all Dimensions.
 
 The parameters mentioned above are specified using the following syntax:
 
-    protocol://ws-entry-point/resource/flowRef/key/providerRef/componentId
-    
-Furthermore, some keywords may be used:
-
-Keywords | Scope | Description
---- | --- | ---
-all | key | Returns data availability information pertaining to all the data belonging to the specified dataflow and provided by the specified provider.
-all | providerRef | Returns data availability information matching the supplied key and belonging to the specified dataflow that has been provided by any data provider. As `all` is a reserved keyword in the SDMX RESTful API, it is recommended not to use it as an identifier for providers.
-all | componentId | Returns data availability information matching the supplied key and belonging to the specified dataflow that has been provided by the specified data providers for all Dimensions. As `all` is a reserved keyword in the SDMX RESTful API, it is recommended not to use it as an identifier for providers.  As `all` is a reserved keyword in the SDMX RESTful API, it is recommended not to use it as an identifier for Dimension Ids.
+    protocol://ws-entry-point/resource/context/agencyID/resourceID/version/key/componentId
 
 The following rules apply:
 
-- If no key is specified, it is equivalent to using the keyword `all`.
-- If no providerRef is specified, it equivalent to using the keyword `all`.
-- If no componentId is specified, it equivalent to using the keyword `all`.
+- The `version`, `key` and `componentId` parameters offer the option to retrieve all existing values, using the `*` operator. This is the default value for these parameters, unless specified otherwise.
```

Comment:
Maybe reformulate to: "To retrieve the availability for all dimensions for unfiltered data, for any agency and for any version, the `*` operator can be used for `componentId`, `key`, `agencyID` and `version`. These are the default values for these parameters, unless specified otherwise."  
About this last sentence, what would be the "default value" for agencyID, if it wasn't '*'?

Reply:
I'm not sure `agencyID` should allowed to be `*` for availability queries. @agent96, what do you think? If it should be allowed, should this also be allowed for the `resourceID` parameter then? Thanks!

Reply:
Yes, I think wildcarding resource Id would be very useful - to be able to ask what data is available, for say a country, without having to first know the contex of the domain, I think this would be very beneficial (in fact, we have already extended the API to support this).

Reply:
Thanks, @agent96 & @dosse. I have added agencyID and resourceID to the list of items that can be wildcarded.